doc/nrf/ug_nrf5340.rst

```diff
@@ -0,0 +1,159 @@
+
+For network communication, Zephyr provides a sample that implements a Bluetooth Low Energy controller.
+
+This sample must be programmed to the network core to run any Bluetooth Low Energy samples on nRF5340.
+
+You might need to adjust the Kconfig configuration of this sample to make it compatible with the peer application.
+For example, :option:`zephyr:CONFIG_BT_MAX_CONN` must be equal to the maximum number of connections supported by the application sample.
+
+
+Application samples
+===================
+
+The |NCS| provides a series of :ref:`Bluetooth Low Energy samples <ble_samples>`, in addition to the :ref:`Bluetooth samples in Zephyr <zephyr:bluetooth-samples>`.
+Most of these samples should run on the nRF5340 PDK, but not all have been thoroughly tested.
+Samples that use non-standard features of the Bluetooth Low Energy controller, like the :ref:`ble_llpm` sample, are not supported.
+
+.. Some samples require configuration adjustments to the :ref:`zephyr:bluetooth-hci-rpmsg-sample` sample as described in the `Network sample`_ section.
+
+Some samples require configuration adjustments to the network sample as described in the `Network sample`_ section.
+For instance, you need to modify network sample configuration to make it compatible with :ref:`ble_throughput` sample:
+
+CONFIG_BT_CTLR_TX_BUFFER_SIZE=251
+CONFIG_BT_CTLR_DATA_LENGTH_MAX=251
+
+This set of configuration guarantees that network sample is able to handle BLE DLE update procedure, which is used in the :ref:`ble_throughput` application.
+
+These samples must be programmed to the application core, in the secure domain.
+
+
+Building and programming a sample
+*********************************
+
+You must program both the network sample and one of the supported :ref:`Bluetooth Low Energy samples <ble_samples>`.
+
+Build and program both samples separately by following the instructions in :ref:`gs_programming_ses`.
+Make sure to use ``nrf5340_dk_nrf5340_cpunet`` as board name when building the network sample, and ``nrf5340_dk_nrf5340_cpuapp`` when building the application sample.
+
+.. important::
+   When programming the samples from |SES|, you might get an error message stating that the target cannot be identified.
+   In this case, you can either modify your |SES| installation and projects to add support for programming nRF5340, or program the generated HEX files from the command line instead.
+
+   See the following sections for more information.
+
+
+Adding support for programming nRF5340 in SES
+=============================================
+
+The current version of the Nordic Edition of |SES| (v4.20a) does not include SEGGER J-Link version v6.54c.
+However, this J-Link version is required to program nRF5340 devices.
+
+To add support for programming nRF5340 in |SES|, complete the following steps:
+
+1. Download and install the latest `J-Link Software and Documentation Pack`_ (v6.54c or later).
```

Comment:
SES v4.30a uses J-Link version v6.54b by default. This version is good enough and the user doesn't need to change for v6.54c (tried on Linux)

The user still needs to set Target Device value as indicated in these instructions.

Reply:
It works only with Application CPU

Reply:
There's no decision yet on the release of 4.30, so let's leave as is for now.

Reply:
> 
> 
> It works only with Application CPU

So you cannot program the network core with SES?